src/main/java/com/google/sps/TripDay.java

```diff
@@ -76,4 +101,48 @@ public String getDestination() {
     locationsCopy.addAll(this.locations);
     return locationsCopy;
   }
+
+  /**
+   * Add locations
+   */
+  public void addLocations(ArrayList<String> locations) {
+    this.locations = locations;
+  }
```

Comment:
Just to clarify, I meant the method `addLocation(...)` that would add the location one-by-one (and accept a `String`), but if this works as well then that should be fine.

Reply:
Eshika wil lbe adding the locations all together after she calculates the order so an `addLocation()` function isnt needed

Reply:
Ok gotcha, yeah that sounds good then!